BridgeBidder/Constraints/HasShownSuit.cs

```diff
@@ -15,14 +15,29 @@ public override bool Conforms(Call call, PositionState ps)
         {
             if (GetSuit(_suit, call) is Suit suit)
             {
-                var strain = Call.SuitToStrain(suit);
-                if (_eitherPartner) {
+                var strain = suit.ToStrain();
+                if (_eitherPartner)
+                {
                     return ps.PairState.Agreements.Strains[strain].Shown;
                 }
                 return ps.PairState.Agreements.Strains[strain].LongHand == ps;
             }
             Debug.Fail("No suit for call in HasShownSuit constraint.");
             return false;
         }
+
+        public string Describe(Call call, PositionState ps)
```

Comment:
I think I'd prefer to leave this one for just debugging purposes. For descriptions I also want to avoid using NSEW notation but it's probably fine for a debug log.

Reply:
Though if this could be updated to use relative positions that'd be even better. Even then I think I'd prefer to keep it only for debugging for now.

Reply:
Also, I noticed this seemed to have a somewhat redundant log in at least one scenario. For a 1D-X-2D auction, I got a condition that stated "6-9 points, 8+ pair ♦, Partner N has shown ♦, NS has shown ♦"